applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -67,13 +73,36 @@ def _create_solution_scheme(self):
         elif(scheme_type == "bossak"):
             damp_factor_m = self.dynamic_settings["damp_factor_m"].GetDouble()
             mechanical_scheme = KratosMultiphysics.ResidualBasedBossakDisplacementScheme(damp_factor_m)
+        elif("bdf" in scheme_type or scheme_type == "backward_euler"):
```

Comment:
note that Backward Euler is also known as BDF1

Reply:
Yes, if the name introduced is "bdf1" will execute the Euler